Library/AppEnvironment.swift

```diff
@@ -295,24 +296,26 @@ public struct AppEnvironment {
                                                    ubiquitousStore: KeyValueStoreType,
                                                    userDefaults: KeyValueStoreType) {
 
-    let data: [String:AnyObject?] = [
```

Comment:
we were way too overzealous with our use of `AnyObject` and swift 3 now heavily prefers `Any`. it's kinda wreaked havoc on our code, but def for the better.